scripts/simulation_client.c

```diff
@@ -15,6 +15,9 @@ bool left_wall;
 bool front_wall;
 bool right_wall;
 
+extern uint8_t walls[];
```

Comment:
I don't like the globals, but ok for the client :)

Reply:
:sweat_smile: